unix/vncpasswd/vncpasswd.cxx

```diff
@@ -37,6 +37,15 @@
 
 #include <termios.h>
 
+#include <pwquality.h>
+#include <locale.h>
+#include <libintl.h>
```

Comment:
Please split adding translation support as a separate commit. It makes it easier to see what parts are associated.

Reply:
Okay, it really should be separated.

Reply:
sorry，
Is it adding a new commit in the current PR. 
Or create a new PR to handle translation.

Reply:
It can be part of this PR as long as it's a separate commit.